index.bs

```diff
@@ -149,6 +149,12 @@ If the console is not open when the printer operation is called,
 implementations should buffer messages to show them in the future up to an
 implementation-chosen limit (typically on the order of at least 100).
 
+<h4 id="object-formats">Common Object Formats</h4>
+
+Typically objects will be printed in a format that is suitable for their context. This section describes common ways in which objects are formatted to be most useful in their context.
+
+An object with <dfn>generic JavaScript object formatting</dfn> is a potentially expandable representation of a generic JavaScript object. An object with <dfn>optimally useful formatting</dfn> is an implementation-specific, potentially-interactive object judged to be maximally useful and informative.
```

Comment:
I'd go with "representation of an object" instead of "object" for the last clause here.